src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocNoWsBeforeDescriptionInJavadocTags.java

```diff
@@ -19,24 +20,21 @@ public class InputAbstractJavadocNoWsBeforeDescriptionInJavadocTags implements S
      * href="https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#serial">
      * oracle docs</a> Syntax: <b>@serial field-description | include | exclude</b>
      */
-    // violation above 'Javadoc comment at column 23 has parse error.
-    // Details: mismatched input 'd' expecting <EOF> while parsing JAVADOC'
```

Comment:
please restore all such "details" comments, they are useful for humans who read this file

Reply:
done